upgrade/helpers/docker.py

```diff
@@ -9,13 +9,13 @@
 
 
 def generate_satellite_docker_clients_on_rhevm(
-        client_os, clients_count, ak=None):
+        client_os, clients_count, custom_ak=None):
```

Comment:
I hope direct this function isnt called with parameter name as 'ak' else you have to change in all that instances. At least I checked in clients.py and upgrade_tests/helpers/scenarios.py and direct parameter name is not being used.

Check elsewhere if I am missing. Or better keep it as ak only.

Reply:
Checked, function is not called with keyword 'ak' throughout the code